src/intersectables/boundingBox.cpp

```diff
@@ -0,0 +1,39 @@
+#include "boundingBox.h"
+#include <algorithm>
+#include <limits>
+
+BoundingBox::BoundingBox(const Vector3f& bottomLeft, const Vector3f& topRight)
+    : bottomLeft(bottomLeft), topRight(topRight) {}
+HitRecord* BoundingBox::intersect(const Ray& ray) const { return new HitRecord(); }
+
+BoundingBox BoundingBox::buildFromVectors(const std::vector<Vector3f>& vectors) {
```

Comment:
Why not have a 

Vector3f min = {3x numeric limits max}

Then below
min.x = std::min(other.x, min.x)

Reply:
Still applies.